packages/tools/src/tools/segmentation/strategies/stack/eraseCircle.ts

```diff
@@ -0,0 +1,18 @@
+import type { Types } from '@cornerstonejs/core';
```

Comment:
I'm pretty sure you don't need to refactor this either. You can just branch out the logic for the 'stack' vs 'volume' in the current files. Please don't do refactoring if not necessary